frontend/src/layout/navigation/ProjectSwitcher.tsx

```diff
@@ -109,7 +109,6 @@ function OtherProjectButton({ team }: { team: TeamBasicType }): JSX.Element {
             title={`Switch to project ${team.name}`}
             status="stealth"
             fullWidth
-            disabled={!team.effective_membership_level}
```

Comment:
Note for reviewer: This was already handled in the endpoint, we don't list projects user does not have access to!